package/MDAnalysis/core/groups.py

```diff
@@ -1039,7 +1039,7 @@ def wrap(self, compound="atoms", center="com", box=None):
             if not all(s == 0.0):
                 o.atoms.translate(s)
 
-    def groupby(self, *topattrs):
+    def groupby(self, topattrs):
```

Comment:
Why did you change that? Having `topattrs` as a star argument was allowing not to break existing code.

Reply:
The problem was that, while it still worked when using a single argument, when providing a list of two or more arguments, topattrs becomes a tuple where the first element is a list, and the second is empty. I'm sure there's a better way of doing this

Reply:
See https://github.com/MDAnalysis/mdanalysis/pull/1846#discussion_r177389219

But the current version shouldn't break existing code

Reply:
OK I missed @kain88-de's comment.